### PR TITLE
feat(currency-converter): implementing currency converter with real-time rate fetching

### PR DESCRIPTION
**Pull request by Sabbir2809**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

# Screenshots or example output

### CODE REVIEW

File: src/plays/currencyconverter/Readme.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+- Fetch: Get exchange rates from an online service.
+- Async/Await: Wait for exchange rate data to be ready before using it.
+- API: Special online service providing exchange rates.
+- Tailwind: Make your app look nice without hard-coding styles.
+- Custom Hooks: Reuse code for common tasks like fetching data.
+
+## Consideration
+
+N/A
+
```
Comment (priyankarpal) [inline src/plays/currencyconverter/Readme.md:30]: remove this portion

File: src/plays/currencyconverter/styles.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.bg-img {
```
Comment (priyankarpal) [inline src/plays/currencyconverter/styles.css:1]: Please rename the class to a unique name so the website does not break.

**Review by priyankarpal — CHANGES_REQUESTED** [review]
there is an ui issue please fix it 

Screenshot_2024-05-08_15-08-42

File: src/plays/currencyconverter/styles.css (hunk 2 above)
Comment (Sabbir2809) [reply src/plays/currencyconverter/styles.css]: resolve ui issue with others

**Comment by Sabbir2809** [comment]
fix(currencyConverter): resolve ui issue

**Review by priyankarpal — APPROVED** [review]

**Pull request merged**